domains/certificates/Shared/DataContext/Migrations/20240723091045_ModifiedContractsToUseRecipientIdAndRemovedCertificates.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace DataContext.Migrations
+{
+    /// <inheritdoc />
+    public partial class ModifiedContractsToUseRecipientIdAndRemovedCertificates : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("DELETE FROM public.\"Contracts\";");
```

Comment:
Didnt we agree on a migrations period so we dont delete peoples contracts, we just add recipendId?

Reply:
What do you mean by 'migrations period'?
We agreed to clear the contracts so that I can add the recipientId column